src/structures/GuildChannel.js

```diff
@@ -554,22 +569,15 @@ class GuildChannel extends Channel {
     return invites;
   }
 
-  /* eslint-disable max-len */
+  /**
+   * Options used to clone a guild channel.
+   * @typedef {GuildCreateChannelOptions} GuildChannelCloneOptions
```

Comment:
Is it clone or create? And if it’s clone, where did the other properties go?

Reply:
Clone. Those are inherited from the `GuildCreateChannelOptions` typedef.

Reply:
But this is that typedef, am I missing something?

Reply:
This is `GuildChannelCloneOptions` typedef which inherits props from the typedef that is inside the `{}` on the left and is defined [here](https://github.com/discordjs/discord.js/blob/9dd3a9a018e0c13ba448f6e579bc866d68c75dc1/src/managers/GuildChannelManager.js#L63)

Reply:
Ohhh yeah right my bad lol